src/ugene_version.pri

```diff
@@ -12,7 +12,7 @@ U2_DISTRIBUTION_INFO=sources
 }
 
 # int version levels for executables
-UGENE_VER_MAJOR=38
+UGENE_VER_MAJOR=39
```

Comment:
This file should not be changed - looks like it is some merge issue.

The patch is is good, I will approve it once you remove this file from the patch.